packages/app/src/app/pages/Sandbox/Editor/Content/Tabs/Tab/elements.ts

```diff
@@ -0,0 +1,88 @@
+import styled, { css } from 'styled-components';
+import CloseIcon from 'react-icons/lib/go/x';
+
+export const Container = styled.div`
+  ${({
+    active,
+    dirty,
+    isOver,
+    theme,
+  }: {
+    active: boolean;
+    dirty: boolean;
+    isOver: boolean;
+    theme: any;
+  }) => css`
+    position: relative;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    line-height: 1;
+    height: calc(100% - 1px);
+    font-size: 0.875rem;
+    cursor: pointer;
+
+    border-bottom: 1px solid transparent;
+
+    padding: 0 1rem;
+    padding-left: 0.75rem;
+    padding-right: 0.125rem;
+    color: ${theme[`tab.inactiveForeground`] ||
+      (theme.light ? css`rgba(0, 0, 0, 0.5)` : css`rgba(255, 255, 255, 0.5)`)};
+
+    svg {
+      font-size: 1rem;
+      margin-right: 0.5rem;
+    }
+
+    background-color: ${theme[`tab.inactiveBackground`] || css`transparent`};
+
+    ${isOver &&
+      css`
+        background-color: ${theme.background2.lighten(0.2)};
+      `};
+    ${active &&
+      css`
+        background-color: ${theme[`tab.activeBackground`] || theme.background2};
+        border-color: ${theme.secondary};
+        color: ${theme[`tab.activeForeground`] ||
+          theme[`editor.foreground`] ||
+          css`white`};
+      `};
+    ${dirty &&
+      css`
+        font-style: italic;
+      `};
+  `}
+`;
+
+export const TabTitle = styled.div`
+  padding-right: 0.5rem;
+  padding-left: 6px;
+  white-space: nowrap;
+`;
+
+export const TabDir = styled.div`
+  color: rgba(255, 255, 255, 0.3);
+  padding-right: 0.5rem;
+  white-space: nowrap;
+`;
+
+export const StyledCloseIcon = styled(CloseIcon)`
```

Comment:
@Saeris You can define the props like this,  so you don't have to specify the `theme` type.

```ts
export const StyledCloseIcon = styled(CloseIcon)<Props>`
  // ...
`;
```